modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -3274,7 +3313,7 @@ protected GridCommunicationClient createTcpClient(ClusterNode node, int connIdx)
 
             long needWaitDelay0 = 200;
```

Comment:
Initial wait delay and max delay below (60_000) are hardcoded but deserve their own (documented) constants.